tests/datavolume_test.go

```diff
@@ -892,10 +941,16 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				dataVolume = utils.NewDataVolumeWithHTTPImportToBlockPV(dataVolumeName, "1G", url(), f.BlockSCName)
 			case "import-vddk":
 				dataVolume = createVddkDataVolume(dataVolumeName, "1Gi", vcenterURL())
-				utils.ModifyDataVolumeWithVDDKImportToBlockPV(dataVolume, f.BlockSCName)
+				utils.ModifyDataVolumeWithImportToBlockPV(dataVolume, f.BlockSCName)
```

Comment:
Any reason all of these are using the block storage class?

Reply:
I think that's just what this test section is for: `Describe("Verify DataVolume with block mode",`

It seems like a good idea to test multi-stage imports with block and file volumes, I know I had a problem with this in the past.

Reply:
Yeah, that was my thought as well, in general block is the more problematic of the two but better that have some testing on both. You can add some FS tests in another PR.